phabfive/cli.py

```diff
@@ -78,13 +78,20 @@
 sub_paste_args = """
 Usage:
     phabfive paste list
+    phabfive paste create <title> input <file> [options]
     phabfive paste show <ids> ... [options]
 
 Arguments:
     <ids> ...            Paste monogram (P123), example P1 P2 P3
+    <title>              Title for Paste
+    <file>               A file with text content for Paste ex. myfile.txt
 
 Options:
     -h, --help           Show this help message and exit
+
+Paste Create Options:
+    -t, --tags=<args> ...           Project name(s), ex. --tags=projectX,projectY,projectZ
+    -s, --subscribers=<args> ...    Subscribers - user, project, mailing list name. Ex --subscribers=user1,user2,user3
```

Comment:
`<args>` is a very programmer-esque term. Could you use `tagX,projectY` or similar?